package.json

```diff
@@ -2,11 +2,12 @@
   "name": "sugar-env",
   "version": "1.5.2",
   "description": "Sugar code to read environment variables with default values.",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
     "coverage": "nyc --branches 100 --functions 100 --lines 100 --statements 100 npm test",
     "lint": "standard --global describe --global it",
-    "test": "mocha test/*.test.js --check-leaks --full-trace --use_strict --recursive"
+    "test": "mocha test/*.test.js --check-leaks --full-trace --use_strict --recursive",
+    "build": "tsc"
```

Comment:
There should also be a `prepare` script that runs `tsc` so we don't forget to transpile before publishing

Reply:
Solved!